Dockerfile

```diff
@@ -0,0 +1,157 @@
+  && apt install libfontconfig \
+  && curl --silent --show-error --location --fail --retry 3 --output /tmp/phantomjs.tar.bz2 ${PHANTOMJS_URL} \
+  && tar -x -C /tmp -f /tmp/phantomjs.tar.bz2 \
+  && mv /tmp/phantomjs-*-linux-x86_64/bin/phantomjs /usr/local/bin \
+  && rm -rf /tmp/phantomjs.tar.bz2 /tmp/phantomjs-* \
+  && phantomjs --version
+
+# install firefox
+
+# If you are upgrading to any version newer than 47.0.1, you must check the compatibility with
+# selenium. See https://github.com/SeleniumHQ/selenium/issues/2559#issuecomment-237079591
+
+RUN FIREFOX_URL="https://s3.amazonaws.com/circle-downloads/firefox-mozilla-build_47.0.1-0ubuntu1_amd64.deb" \
+  && curl --silent --show-error --location --fail --retry 3 --output /tmp/firefox.deb $FIREFOX_URL \
+  && echo 'ef016febe5ec4eaf7d455a34579834bcde7703cb0818c80044f4d148df8473bb  /tmp/firefox.deb' | sha256sum -c \
+  && dpkg -i /tmp/firefox.deb || apt -f install  \
+  && apt install -y libgtk3.0-cil-dev libasound2 libasound2 libdbus-glib-1-2 libdbus-1-3 \
+  && rm -rf /tmp/firefox.deb \
+  && firefox --version
+
+# install geckodriver
+
+RUN export GECKODRIVER_LATEST_RELEASE_URL=$(curl https://api.github.com/repos/mozilla/geckodriver/releases/latest | jq -r ".assets[] | select(.name | test(\"linux64\")) | .browser_download_url") \
+      && curl --silent --show-error --location --fail --retry 3 --output /tmp/geckodriver_linux64.tar.gz "$GECKODRIVER_LATEST_RELEASE_URL" \
+      && cd /tmp \
+      && tar xf geckodriver_linux64.tar.gz \
+      && rm -rf geckodriver_linux64.tar.gz \
+      && mv geckodriver /usr/local/bin/geckodriver \
+      && chmod +x /usr/local/bin/geckodriver \
+      && geckodriver --version
+
+# install chrome
+
+RUN curl --silent --show-error --location --fail --retry 3 --output /tmp/google-chrome-stable_current_amd64.deb https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb \
+      && (dpkg -i /tmp/google-chrome-stable_current_amd64.deb || apt -fy install)  \
+      && rm -rf /tmp/google-chrome-stable_current_amd64.deb \
+      && sed -i 's|HERE/chrome"|HERE/chrome" --disable-setuid-sandbox --no-sandbox|g' \
+           "/opt/google/chrome/google-chrome" \
+      && google-chrome --version
+
+RUN export CHROMEDRIVER_RELEASE=$(curl --location --fail --retry 3 http://chromedriver.storage.googleapis.com/LATEST_RELEASE) \
+      && curl --silent --show-error --location --fail --retry 3 --output /tmp/chromedriver_linux64.zip "http://chromedriver.storage.googleapis.com/$CHROMEDRIVER_RELEASE/chromedriver_linux64.zip" \
+      && cd /tmp \
+      && unzip chromedriver_linux64.zip \
+      && rm -rf chromedriver_linux64.zip \
+      && mv chromedriver /usr/local/bin/chromedriver \
+      && chmod +x /usr/local/bin/chromedriver \
+      && chromedriver --version
+
+# start xvfb automatically to avoid needing to express in circle.yml
+ENV DISPLAY :99
```

Comment:
we can remove all the circleci stuff in the dev machines

Reply:
Yes. @jellegerbrandy